src/app/pages/common/ix-forms/components/ix-input/ix-input.component.ts

```diff
@@ -75,15 +73,7 @@ export class IxInputComponent implements ControlValueAccessor {
   }
 
   shouldShowResetInput(): boolean {
```

Comment:
Method is probably not needed.

Reply:
![image](https://user-images.githubusercontent.com/22980553/149951645-f7484502-af03-4f06-9cc1-6f3c064ef2a5.png)
 I think it is needed for reset input.